static/app/views/dashboardsV2/datasetConfig/errorsAndTransactions.tsx

```diff
@@ -57,6 +64,21 @@ export const ErrorsAndTransactionsConfig: DatasetConfig<
   transformTable: transformEventsResponseToTable,
 };
 
+function getEventsTableFieldOptions(
+  contextualProps?: ContextualProps,
+  tags?: TagCollection
+) {
+  const organization = contextualProps?.organization!;
```

Comment:
Nit pick, but
```suggestion
  const {organization} = contextualProps!;
```
?

Reply:
I'll still need to do `organization!` in the references to organization below, figured i might as well do this when i de-structure?